mars/services/scheduling/supervisor/assigner.py

```diff
@@ -101,3 +115,24 @@ async def assign_subtasks(self, subtasks: List[Subtask]):
                         bands.append(band)
             assigns.append(random.choice(bands))
         return assigns
+
+    def reassign_band(self):
+        assert self._bands
+        return random.choice(self._bands)
+
+    async def reassign_subtasks(self, band_num_queued_subtasks: Dict[Tuple, int]) -> Dict[Tuple, int]:
+        used_bands = band_num_queued_subtasks.keys()
+        # select available bands which may contain new available unused ones
+        bands_to_assign = [used_band for used_band in used_bands if used_band in self._bands]
```

Comment:
Why not use all bands? If we have only one band initially, we can never rebalance those bands to other bands